src/components/Contacts/ContactDetails/ContactDetailsTab/People/ContactDetailsTabPeople.tsx

```diff
@@ -0,0 +1,136 @@
+  width: '100%',
+}));
+
+const ContactPersonTextContainer = styled(Box)(() => ({
+  margin: 0,
+  flexGrow: 4,
+  marginBottom: '10px',
+}));
+
+const ContactPersonRowContainer = styled(Box)(({ theme }) => ({
+  margin: theme.spacing(1),
+  display: 'flex',
+  alignItems: 'center',
+}));
+
+const ContactPersonPrimaryText = styled(Typography)(({ theme }) => ({
+  margin: theme.spacing(0, 1),
+  color: theme.palette.text.hint,
+}));
+
+const ContactPersonIconContainer = styled(Box)(() => ({
+  width: '18px',
+  height: '18px',
+  marginRight: '35px',
+}));
+
+interface ContactDetailsPeopleProp {
+  data: ContactDetailsTabQuery;
+}
+
+export const ContactDetailsTabPeople: React.FC<ContactDetailsPeopleProp> = ({
+  data,
+}) => {
+  const { t } = useTranslation();
+
+  const personView = (person: Person, isPrimary: boolean) => {
+    return (
+      <ContactPersonContainer>
+        <ContactPersonAvatar
+          alt={`${person.firstName} ${person.lastName}`}
+          src={person.lastName}
+        />
+        <ContactPersonTextContainer>
+          {/* Heading Section */}
+          <ContactPersonRowContainer>
+            <Typography variant="h6">
+              {`${person.firstName} ${person.lastName}`}
+            </Typography>
+            {isPrimary ? (
+              <ContactPersonPrimaryText variant="subtitle1">
+                {`- ${t('Primary')}`}
+              </ContactPersonPrimaryText>
+            ) : null}
+          </ContactPersonRowContainer>
+          {/* Phone Number */}
+          {person.primaryPhoneNumber !== null ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Phone color="disabled" />
+              </ContactPersonIconContainer>
+              <Typography variant="subtitle1">
+                {person.primaryPhoneNumber.number}
+              </Typography>
+              <Typography variant="caption">
+                {` - ${person.primaryPhoneNumber.location}`}
+              </Typography>
+            </ContactPersonRowContainer>
+          ) : null}
+          {/* Email Section */}
+          {person.primaryEmailAddress !== null ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Email color="disabled" />
+              </ContactPersonIconContainer>
+              <Typography variant="subtitle1">
+                {person.primaryEmailAddress.email}
+              </Typography>
+            </ContactPersonRowContainer>
+          ) : null}
+          {/* Birthday Section */}
+          {person.birthdayDay !== null ? (
+            <ContactPersonRowContainer>
+              <ContactPersonIconContainer>
+                <Cake color="disabled" />
+              </ContactPersonIconContainer>
+              {/* TODO: Change to local format for different countries */}
+              <Typography variant="subtitle1">
+                {`${person.birthdayMonth}/${person.birthdayDay}/${person.birthdayYear}`}
```

Comment:
And then I'm trying to figure out what to do with the year in case it's null. Do we just hide it and render ```MM//dd``` or do we do something like ```MM/dd/NA``` or default the year like the rest api does and do ```MM/dd/1900```? Let me know what you think.